jwql/utils/engineering_database.py

```diff
@@ -0,0 +1,174 @@
+#! /usr/bin/env python
+"""Module to interface the JWST Engineering Database.
+
+This module provides ``jwql`` with functions to interface and query the
+ JWST DMS Engineering Database.
+
+Authors
+-------
+
+    - Johannes Sahlmann
+
+Use
+---
+
+    This module can be imported and used with
+
+    ::
+
+        from jwql.engineering_database import query_single_mnemonic
+        query_single_mnemonic(mnemonic_identifier, start_time, end_time)
+
+    Required arguments:
+
+    ``mnemonic_identifier`` - String representation of a mnemonic name.
+    ``start_time`` - astropy.time.Time instance
+    ``end_time`` - astropy.time.Time instance
+
+Notes
+-----
+    This module is built on top of astroquery.mast and uses JWST-specific MAST services.
+    The user has to provide a valid MAST authentication token.
+
+"""
+import astropy
+from astropy.table import Table
+from astroquery.mast import Mast
+
+from .utils import get_config
+
+# should use oauth.register_oauth()?
+settings = get_config()
+mast_token = settings['mast_token']
+
+Mast.login(token=mast_token)
+
+# could eventually be moved to constants.py
+mast_edb_timeseries_service = 'Mast.JwstEdb.GetTimeseries.All'
+mast_edb_dictionary_service = 'Mast.JwstEdb.Dictionary'
+mast_edb_mnemonic_service = 'Mast.JwstEdb.Mnemonics'
+
+
+class EdbMnemonic():
+    """Class to hold and manipulate results of EngDB queries."""
+
+    def __init__(self, mnemonic_identifier, start_time, end_time, data, meta, info):
+        """Populate attributes."""
```

Comment:
@Johannes-Sahlmann Could you add a `Parameters` section to the docstring to describe all of the parameters?

Reply:
Done. 
Do you have an automated way of checking for such omissions? I am using a number of PEP-compliance checkers but none has picked that up.